pkgs/dartpad_ui/test/autosave_test.dart

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:dartpad_ui/local_storage.dart';
+import 'package:dartpad_ui/model.dart';
+import 'package:dartpad_ui/samples.g.dart';
+import 'package:dartpad_ui/utils.dart';
+
+import 'package:test/test.dart';
+
+String getFallback() =>
+    LocalStorage.instance.getUserCode() ?? Samples.defaultSnippet();
+
+Never throwingFallback() =>
+    throw StateError('DartPad tried to load the fallback');
+
+void main() {
+  const channel = Channel.stable;
+  group('Autosave:', () {
```

Comment:
I would take the colon out since it's not doing anything useful. The test package formats the output.